packages/bsky/src/ingester/label-subscription.ts

```diff
@@ -0,0 +1,73 @@
+import AtpAgent from '@atproto/api'
+import { PrimaryDatabase } from '../db'
+import { sql } from 'kysely'
+import { dbLogger } from '../logger'
+import { SECOND } from '@atproto/common'
+
+export class LabelSubscription {
```

Comment:
Added this as part of the appview ingester. You can imagine it consuming the `subscribeLabels` stream, but since we don't have that yet, it "subscribes" to labels by repeatedly calling `temp.fetchLabels`